tests/erlang_tests/test_binary_to_term.erl

```diff
@@ -25,17 +25,29 @@
 start() ->
     % Starting from OTP-26, atoms are encoded as UTF-8 by default.
     TermToBinaryOptions =
-        case erlang:system_info(machine) of
-            "BEAM" ->
-                case erlang:system_info(version) >= "13.2" of
-                    true -> [{minor_version, 1}];
-                    false -> []
-                end;
-            "ATOM" ->
+        case get_otp_version() of
+            X when is_integer(X) andalso X >= 26 ->
+                [{minor_version, 1}];
+            atomvm ->
+                [{minor_version, 1}];
+            _ ->
                 []
         end,
     test_reverse(foo, <<131, 100, 0, 3, 102, 111, 111>>, TermToBinaryOptions),
     test_reverse(bar, <<131, 100, 0, 3, 98, 97, 114>>, TermToBinaryOptions),
+    test_reverse(foo, {<<131, 119, 3, 102, 111, 111>>, <<131, 100, 0, 3, 102, 111, 111>>}),
```

Comment:
None of the following tests is making use of the newly introduced tag `ATOM_UTF8_EXT` (that is `118`), they are all using `SMALL_ATOM_UTF8_EXT` that is `119`.
E.g. `<<131, 119, ...`.

Reply:
Correct.